src/rspq/rspq.c

```diff
@@ -506,15 +515,22 @@ static void rspq_crash_handler(rsp_snapshot_t *state)
 static void rspq_assert_invalid_command(rsp_snapshot_t *state)
 {
     rsp_queue_t *rspq = (rsp_queue_t*)state->dmem;
+
+    // Get overlay index a name. Be defensive against DMEM corruptions.
+    int ovl_idx = rspq->current_ovl / sizeof(rspq_overlay_t);
+    const char *ovl_name = "?";
+    if (ovl_idx < RSPQ_MAX_OVERLAY_COUNT && rspq_overlay_ucodes[ovl_idx])
+        ovl_name = rspq_overlay_ucodes[ovl_idx]->name;
```

Comment:
This could go into a helper function as it's a duplicate from the crash handler.

Reply:
Done